f2/src/locales/fr.json

```diff
@@ -208,7 +208,7 @@
   "howDidItStartPage.incidentInformation": "Informations sur l'incident",
   "howDidItStartPage.intro": "N’importe qui peut être victime d’une fraude ou d’un crime informatique. Les cybercriminels utilisent plusieurs techniques pour réussir à obtenir ce qu’ils veulent.",
   "howDidItStartPage.nextButton": "Continuer",
-  "howDidItStartPage.nextPage": "Étape suivante : D’après vous, quelles pourraient être les répercussions?",
+  "howDidItStartPage.nextPage": "Étape suivante : Quand est-ce que l'incident s'est produit?",
```

Comment:
Can  you please use the French translation from the figma :https://www.figma.com/file/R054FzUOrpP13oTbMsQevj/Sprint-1-%26-2-(Beta-2)?node-id=4940%3A523. the translation is 
"howDidItStartPage.nextPage":Étape suivante : Quand l’incident a-t-il eu lieu?

Reply:
Thanks! Updated.